src/angular/lightbox/lightbox-container.ts

```diff
@@ -82,6 +89,7 @@ export class SbbLightboxContainer extends _SbbDialogContainerBase {
     ngZone: NgZone,
     overlayRef: OverlayRef,
     private _changeDetectorRef: ChangeDetectorRef,
+    private _viewportRuler: ViewportRuler,
```

Comment:
Seems like nobody is using the `SbbLightboxContainer` directly. Should we mark it as breaking change anyways?